compiler/sem.nim

```diff
@@ -19,7 +19,8 @@ import
   intsets, transf, vmdef, vm, aliases, cgmeth, lambdalifting,
   evaltempl, patterns, parampatterns, sempass2, linter, semmacrosanity,
   lowerings, plugins/active, lineinfos, strtabs, int128,
-  isolation_check, typeallowed, modulegraphs, enumtostr, concepts, astmsgs
+  isolation_check, typeallowed, modulegraphs, enumtostr, concepts, astmsgs,
+  extccomp
```

Comment:
Remove again?

Reply:
Yes, sorry. Totally forgot about it

Reply:
hmm, do we still want to move the `localPassC` right? If so, is still necessary